# DroneWayPointType
uint8 type
uint8 POINT_TAKEOFF = 0
uint8 POINT_FLYING = 1
uint8 POINT_LANDING = 2

Position pos
float64 v
uint32 timeoutsec